src/main/webapp/index.html

```diff
@@ -51,17 +51,17 @@
       </div>
       <div class="inline">
         <form id="input-form">
-          <input type="text" class="input-text" id="input-text" name="input-text" placeholder="Enter your postcard text.." required="True"> 
+          <input type="text" class="input-text" id="input-text" name="input-text" placeholder="Enter your postcard text.."> 
           <label class="container"> Tick to share location
             <input type="checkbox" id="location-checkbox" name="location-checkbox" onclick="getUserLocation()" value="none">
             <span class="checkmark"></span>
           </label>
           <br/>
         </form>
+        <button class="rec" onclick="record()" id="recording">Record message</button>
```

Comment:
there is a more up-to-date version of this code in dev already

Reply:
this is the most up-to-date, we discussed that with Mara